ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -111,6 +111,13 @@ VERSION=$(oc adm release info ${TESTING_RELEASE_IMAGE} --output=json | jq -r '.m
 
 set -o errexit
 
+# Ensure at least 3 control planes for vSphere. Single node is not supported.
```

Comment:
you will probably need to update ci-operator/step-registry/ipi/conf/vsphere/vcm/ipi-conf-vsphere-vcm-commands.sh as well. we are deprecating this step and its possible your workflow might not even be using it.

Reply:
addressed in https://github.com/openshift/release/commit/d2eff0b8b1d255db8ad65eaef7bc7d8924756a14

Reply:
@rvanderp3   Windows jobs in vSphere are using  [ ipi-vsphere-ovn-hybrid-custom-vxlan-port](https://github.com/openshift/release/blob/27b93c234e0857c07ce8b8b8f3ce48634ecd3bbf/ci-operator/step-registry/ipi/vsphere/ovn/hybrid-custom-vxlan-port/ipi-vsphere-ovn-hybrid-custom-vxlan-port-workflow.yaml#L2) that relies on `ipi-conf-vsphere`.

are you saying we should move aways from `ipi-conf-vsphere`?

Reply:
That is the chain not the step

https://steps.ci.openshift.org/job?org=openshift-priv&repo=windows-machine-config-operator&branch=release-4.17&test=vsphere-e2e-operator

https://steps.ci.openshift.org/chain/ipi-conf-vsphere

Any job that runs in CI (not devqe) will use https://steps.ci.openshift.org/reference/ipi-conf-vsphere-vcm

Reply:
got it, makes sense.